src/fpm/manifest/profiles.f90

```diff
@@ -0,0 +1,879 @@
+!> Implementation of the meta data for compiler flag profiles.
+!>
+!> A profiles table can currently have the following subtables:
+!> Profile names - any string, if omitted, flags are appended to all matching profiles
+!> Compiler - any from the following list, omitting it yields an error
```

Comment:
Need to add a blank comment line here for FORD to render the bullet points correctly. Also below.
```suggestion
!> Compiler - any from the following list, omitting it yields an error
!>
```